### ISSUE TITLE
убиваем частицу ни внутри

### ISSUE THREAD

**Issue by veramal** [tuffnatty/suddenly#208]
Решили убрать из описания Part _ни_, т.к. она пишется отдельно во всех случаях, кроме пары примеров в грамматике. В таблице вычеркнула, правило 32 убрала, больше она нигде не упоминается.

**Comment by tuffnatty** [comment]
Если так сделать, перестают разбираться полдыним, иттіним из #81.

**Comment by veramal** [comment]
Да, это понятно. Жаль, тут не написано, чем она мешает, надо вспомнить.

**Comment by tuffnatty** [comment]
тесты (см выше) по-прежнему сломаны

**Comment by veramal** [comment]
Кажется, это надо откатить пока.

**Comment by tuffnatty** [comment]
Откатил

**Comment by veramal** [comment]
Нашелся еще один пример со слитным написанием (пусть и через _е_ вместо _и_):  _Ноға ла ӧдіртінем?_ 'Зачем только я приказал его убить?' (Хара-пайнаң Ӧкіс-оол). Кажется, можно ничего не делать.
